assets/icons.js

```diff
@@ -750,3 +750,29 @@ export const TelegramChannel = ({ color = 'black', width = 20, height = 22 }) =>
     </defs>
   </svg>
 );
+
+export const Close = ({ width = 20, height = 20, color = '#000000' }) => (
```

Comment:
These changes should be implemented in beta. So use beta branch as a base branch

Reply:
https://github.com/Juratbek/upper_frontend/pull/1351